src/main/resources/static/css/charsheet.css

```diff
@@ -13,65 +49,78 @@ body {
 
 #name {
   font-weight: bold;
-	font-size: 30px;
+  font-size: 1.5em;
+}
+
+#nameInputDiv #name {
+   width: 100%;
 }
 
 .avatarContainer {
-  width: 500px;
+  width: 100%;
 }
 
 .avatarContainer img {
   max-height: 100%;
   max-width: 100%;
 }
 
-div.section {
-  border: 1px solid #ccc;
-  padding: 15px;
+div .section {
+  padding: 16px 24px;
+}
+
+/* --- Character Stats --- */
+div .stat {
+  padding-top: 5px;
 }
 
-span.statLabel {
+div .stat p {
+   display: inline-block;
+   font-weight: 300;
+}
+
+span .statLabel {
   display:inline-block;
-  width: 110px;
-  font-weight: bold;
 }
 
-div.stat {
-  padding-top: 5px;
+div .statLabel p {
+   font-weight: 700;
+   width: 40%;
 }
+/* --- Character Stats close --- */
 
 .skillCluster {
   font-weight: bold;
   display: inline-block;
   margin-top: 15px;
 }
 
-span.skillCost, span.specCost {
+span .skillCost, span .specCost {
   font-style: italic;
   color: #888;
   font-size: 14px;
   margin-left: 20px;
 }
 
-span.skillLabel {
+span .skillLabel {
   display: inline-block;
   width: 100px;
 }
 
-span.specLabel {
+span .specLabel {
 	display: inline-block;
 	width: 90px;
 	margin-left: 10px;
 	font-style: italic;
 }
 
-span.skillValue, span.specRanks {
+span .skillValue, span .specRanks {
 	display: inline-block;
 	width: 30px;
 	text-align: right;
 }
 
-input.specName {
+input .specName {
```

Comment:
Before, the CSS rule was "input with class `specName`". Now you have it as "input with a child with class `specName`", which won't ever evaluate.

(This comment applies to the 4 changes above this one, too)

Reply:
Got it. I don't remember why I changed that... 
Changed it back.